qa/common/gen_qa_model_repository

```diff
@@ -197,6 +197,7 @@ ln -s /usr/bin/python3 /usr/bin/python
     apt update && \
     apt install -y intel-openvino-dev-ubuntu20-${OPENVINO_VERSION})
 
+pip3 install "protobuf<=3.20.1"
```

Comment:
Add a comment here and below for this change.

Reply:
Requested change added